subgraph/src/Agreement.ts

```diff
@@ -27,17 +27,21 @@ import {
 
 export function handleSettingChanged(event: SettingChanged): void {
   const agreementApp = AgreementContract.bind(event.address)
-  const settingData = agreementApp.getSetting(event.params.settingId)
+  const settingData = agreementApp.try_getSetting(event.params.settingId)
+
+  if(settingData.reverted){
+    return
```

Comment:
Maybe we can log an error here with some description and the agreement app address, so that we can track this errors?